packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -59,11 +92,15 @@ export function bindPreferenceSchemaProvider(bind: interfaces.Bind): void {
     bindContributionProvider(bind, PreferenceContribution);
 }
 
+export function bindDefaultPreferenceProvider(bind: interfaces.Bind): void {
+    bind(PreferenceProvider).to(PreferenceSchemaProvider).inSingletonScope().whenTargetNamed(PreferenceScope.Default);
```

Comment:
It creates a completely new instance of `PreferenceSchemaProvider`, you should use `toService` or `toDynamicValue` to get an instance of `PreferenceSchemaProvider` bound by `bindPreferenceSchemaProvider`.

Reply:
Thank you for the review, i went through my changes and realized that `bindDefaultPreferenceProvider` is no longer needed. `PreferenceSchemaProvider` is now only injected, and never created by `PreferenceProviderProvider`. So this function is removed.